diff --git a/src/sha256.h b/src/sha256.h
index 77ddb6d..67ccf7a 100644
--- a/src/sha256.h
+++ b/src/sha256.h
@@ -49,7 +49,7 @@ static void sha256_init_h(unsigned int h[8])
     h[7] = 0x5be0cd19U;
 }
 
-static void sha256_init(SHA256_CTX *ctx)
+void sha256_init(SHA256_CTX *ctx)
 {
     sha256_init_h(ctx->h);
     ctx->N = 0;
@@ -57,7 +57,7 @@ static void sha256_init(SHA256_CTX *ctx)
     ctx->off = 0;
 }
 
-static void sha256_update(SHA256_CTX *ctx, const void *_inp, size_t len)
+void sha256_update(SHA256_CTX *ctx, const void *_inp, size_t len)
 {
     size_t n;
     const unsigned char *inp = _inp;
@@ -116,7 +116,7 @@ static void sha256_emit(unsigned char md[32], const unsigned int h[8])
 }
 #endif
 
-static void sha256_final(unsigned char md[32], SHA256_CTX *ctx)
+void sha256_final(unsigned char md[32], SHA256_CTX *ctx)
 {
     unsigned long long bits = ctx->N * 8;
     size_t n = ctx->off;
